modules/selenium/src/main/java/org/testcontainers/containers/RecordingFileFactory.java

```diff
@@ -1,15 +1,16 @@
 package org.testcontainers.containers;
 
 import org.junit.runner.Description;
+import org.testcontainers.containers.VncRecordingContainer.VncRecordingFormat;
 
 import java.io.File;
 
 public interface RecordingFileFactory {
 
     @Deprecated
-    default File recordingFileForTest(File vncRecordingDirectory, Description description, boolean succeeded) {
-        return recordingFileForTest(vncRecordingDirectory, description.getTestClass().getSimpleName() + "-" + description.getMethodName(), succeeded);
+    default File recordingFileForTest(File vncRecordingDirectory, Description description, boolean succeeded, VncRecordingFormat recordingFormat) {
```

Comment:
I would not break existing interfaces and update this method

Reply:
Done, I canceled the changes.
Thanks